kubeflow-katib.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  epoch: 0
+  name: kubeflow-katib
+  version: 0.15.0
+  description: Kubeflow Katib services
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - wget
+      - ca-certificates
```

Comment:
I think this may have come from something I added, having said that I think they can be dropped

Reply:
we can remove this and take the PR off draft

Reply:
ok, cool, thanks!